src/components/Socials/styles.scss

```
@@ -1,3 +1,64 @@
+// Mobile portrait
 .wrapper {
+	display: flex;
+	justify-content: center;
+	width: 100%;
 	font-size: 1em; //nothing comment to make sure class appears
+	
+		li {
+			width: 1em;
+			height: 1em;
+			list-style: none;
+		}
+
+			a {
+				display: block;
+				width: 100%;
+				height: 100%;
+				background-color: blue;
+
+				&[aria-label="Facebook"]{
+					background-image: url("path/to/facebook.jpg");
```

```suggestion
					background-image: url("path/to/facebook.svg");
```
Just me being picky - we know that we need transparency so it'll either be a `svg` or a `png`, and it's a nice simple shape so we'll probably go for `svg`.  All of the 404s that appear in the console will basically become our shopping list of assets in the details stage.